servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/HttpClients.java

```diff
@@ -206,11 +217,16 @@ private HttpClients() {
      * {@link HttpHeaderNames#HOST}. Use {@link SingleAddressHttpClientBuilder#unresolvedAddressToHost(Function)}
      * if you want to override that value or {@link SingleAddressHttpClientBuilder#hostHeaderFallback(boolean)} if you
      * want to disable this behavior.
+     * <p>
+     * Note, if {@link SingleAddressHttpClientBuilder#proxyAddress(Object) a proxy} is configured for this client,
```

Comment:
This information was present in the deprecated methods. Unfortunately, they didn't work even for resolved proxy addresses. This is an attempt to warn the users about the expected usage of the returned builder.